vomsimporter.py

```diff
@@ -680,6 +723,17 @@ def __init__(self, host, port, vo, ldap_host, ldap_port, protocol="https", usern
         if email_mapfile is not None:
             self._load_email_override_csv_file(email_mapfile)
 
+        self._voms_groups = None
+        if voms_groups or voms_roles:
+            self._voms_groups = set()
+            if voms_groups:
+                for g in voms_groups:
+                    self._voms_groups.add(voms2iam_group_name(g))
+                if voms_roles:
+                    for g in voms_groups:
+                        for r in voms_roles:
+                            self._voms_groups.add(voms2iam_group_name(fqan2iam_group_name("{0}/{1}".format(g, r))))
```

Comment:
The call to `voms2iam_group_name` seems superfluous to me: it just removes the initial `/`, which is already done by `fqan2iam_group_name`.

Reply:
I missed that `fqan2iam_group_name` returns result starting at first character...